### PR TITLE
Remove runtime related tests

### PR DESCRIPTION
**Pull request by armaniferrante**
The tests removed here are testing the runtime, not the client library, and so I propose moving them to the more appropriate location.

### CODE REVIEW

File: test/web3c.js [hunk 1]
```diff
@@ -144,36 +133,4 @@ describe('Web3', () => {
     // since the client uses a different ephemeral key each time, it
     // won't always be able to decode the returned log.
   }).timeout(timeout);
-
-  it('should estimate gas for confidential transactions the same as gas actually used', async () => {
```
Comment (willscott) [inline test/web3c.js]: why can't this one be used?

Comment (willscott) [reply test/web3c.js]: (we also want to make sure that the client library internally runs estimate gas and applies it correctly)

Comment (armaniferrante) [reply test/web3c.js]: Agreed we want this. However, we should change the test. 

It shouldn't test that the gas is the same. It should just test that estimateGas returns something. The "sameness" point of this test is really runtime-specific behavior.

Comment (armaniferrante) [reply test/web3c.js]: Updated the test to reflect the client behavior we care about.

**Pull request merged**